src/Json.Schema.ToDotNet/WellKnownMethodNames.cs

```diff
@@ -5,8 +5,11 @@ namespace Microsoft.Json.Schema.ToDotNet
 {
     public static class WellKnownMethodNames
     {
-        public const string EqualsMethod = "Equals";
-        public const string GetHashCodeMethod = "GetHashCode";
-        public const string ReferenceEqualsMethod = "ReferenceEquals";
+        public const string ToList = nameof(ToList);
+        public const string Compare = nameof(Compare);
+        public const string CompareTo = nameof(CompareTo);
+        public new const string Equals = nameof(Equals);
+        public new const string GetHashCode = nameof(GetHashCode);
+        public new const string ReferenceEquals = nameof(ReferenceEquals);
```

Comment:
`new` keyword to overwrite predefined object method names. since we only use this class as constants container.